docs/inventory-plugins.rst

```diff
@@ -8,218 +8,120 @@ Inventory Plugins
 
-.. _ug_source_ec2:
+.. _ag_inv_import:
 
-Amazon Web Services EC2
-~~~~~~~~~~~~~~~~~~~~~~~~
+Inventory File Importing
+-------------------------
 
 .. index::
-   pair: inventories; Amazon Web Services
-
-1. To configure an AWS EC2-sourced inventory, select **Amazon EC2** from the Source field.
+   single: inventory file importing
+   single: inventory scripts; custom
 
-2. The Create Source window expands with additional fields. Enter the following details:
+AWX allows you to choose an inventory file from source control, rather than creating one from scratch. This function is the same as custom inventory scripts, except that the contents are obtained from source control instead of editing their contents browser. This means, the files are non-editable and as inventories are updated at the source, the inventories within the projects are also updated accordingly, including the ``group_vars`` and ``host_vars`` files or directory associated with them. SCM types can consume both inventory files and scripts, the overlap between inventory files and custom types in that both do scripts.
 
-   - **Credential**: Optionally choose from an existing AWS credential. If AWX is running on an EC2 instance with an assigned IAM Role, the credential may be omitted, and the security credentials from the instance metadata will be used instead. For more information on using IAM Roles, refer to the AWS documentation, `IAM roles for Amazon EC2 <http://docs.aws.amazon.com/AWSEC2/latest/UserGuide/iam- roles-for-amazon-ec2.html>`_.
+Any imported hosts will have a description of "imported" by default. This can be overridden by setting the ``_awx_description`` variable on a given host. For example, if importing from a sourced .ini file, you could add the following host variables:
 
-3. You can optionally specify the verbosity, host filter, enabled variable/value, and update options as needed.
+::
 
-4. Use the **Source Variables** field to override variables used by the ``aws_ec2`` inventory plugin. Enter variables using either JSON or YAML syntax. Use the radio button to toggle between the two. For a detailed description of these variables, view the `aws_ec2 inventory plugin documentation <https://cloud.redhat.com/ansible/automation-hub/repo/published/amazon/aws/content/inventory/aws_ec2>`__.
+   [main]
+   127.0.0.1 _awx_description="my host 1"
+   127.0.0.2 _awx_description="my host 2"
 
-|Inventories - create source - AWS EC2 example|
+Similarly, group descriptions also default to "imported", but can be overridden by the ``_awx_description`` as well.
 
-.. |Inventories - create source - AWS EC2 example| image:: _static/images/inventories-create-source-AWS-example.png
```

Comment:
Why is a PR that claims to add a document also reshuffles others? What's the relation?

Also, if the image is no longer included, shouldn't it also be deleted from Git?

Reply:
@webknjaz  can you show me an example of the reshuffling? Yes there will be a separate PR that removes all images from the repo no longer used.

Reply:
Ok I know what you mean, I forgot I dropped the changes in for inventory plugins, that's why I didn't fix the title mismatches or remove the index entries. I can revert the changes for inventory plugins and put them in a separate PR.

Reply:
Having said that, I will proceed with cleaning up the inventory plugins document - fixing the title mismatches, unnecessary index entries and apply the correct roles to both docs, where applicable.

Reply:
Let's not add a bunch of unrelated changes here, though.

Also, didn't we settle on keeping the indexes?

Reply:
No, I have been removing them because we were producing PDFs.

Reply:
I put these back in the last commit per your request.